### PR TITLE
SLE Micro client support matrix page

### PR DESCRIPTION
**Pull request by moio**
# Description

* replaces remaining occurrences of "Service Pack Migration" to "Product Migration" (as we renamed the feature in 4.2)
* rewords the Advanced virtual guest management description for clarity
* **adds a comparison page for SLE Micro supported features**

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

### CODE REVIEW

File: modules/client-configuration/pages/supported-features-sle-micro.adoc [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+| {salt}
+
+| Organization custom states
+| {salt}
+
+| System set manager (SSM)
+| {salt}
+
+| Service pack migration
+| N/A
```
Comment (mcalmer) [inline modules/client-configuration/pages/supported-features-sle-micro.adoc:69]: Should we better say {question} here? Or is somebody trying this out already and we just wait for the result?

File: modules/client-configuration/pages/supported-features-sle-micro.adoc [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+| Basic Virtual Guest Management {star}
+| {cross}
+
+| Advanced Virtual Guest Management {star}
+| {question}
```
Comment (mcalmer) [inline modules/client-configuration/pages/supported-features-sle-micro.adoc:75]: Basic is not working, but advanced should be supported in future? This does not make much sense, or do I miss something?

File: modules/client-configuration/pages/supported-features-sle-micro.adoc [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+| {cross}
+
+| Configuration file management
+| {salt}
+
+| Snapshots and profiles
+| {salt}: Profiles supported, Sync not supported
+
+| Power management
+| {question}
```
Comment (mcalmer) [inline modules/client-configuration/pages/supported-features-sle-micro.adoc:132]: I think Power Management depends more on the Hardware than on the OS.
Only a wrong ACL in the UI could prevent this working. Do we have a problem with the UI ACL?

**Review by mcalmer — COMMENTED** [review]
Some remarks added

File: modules/client-configuration/pages/supported-features-sle-micro.adoc (hunk 1 above)
Comment (moio) [reply modules/client-configuration/pages/supported-features-sle-micro.adoc:69]: Well, SLE Micro does not have Service Packs AFAIU, only major versions. So this should be called differently - do we want to diverge from the feature list of all other OSs?

File: modules/client-configuration/pages/supported-features-sle-micro.adoc (hunk 2 above)
Comment (moio) [reply modules/client-configuration/pages/supported-features-sle-micro.adoc:75]: AFAIU from @cbosdo basic actually means "the set of functionality that the traditional stack provides". Given only Salt is supported here, we could maybe change the title for all OSs. I was confused myself.

File: modules/client-configuration/pages/supported-features-sle-micro.adoc (hunk 3 above)
Comment (moio) [reply modules/client-configuration/pages/supported-features-sle-micro.adoc:132]: Not that I am aware of. We will just need to test this before we remove the "tech preview" label.

File: modules/client-configuration/pages/supported-features-sle-micro.adoc (hunk 1 above)
Comment (mcalmer) [reply modules/client-configuration/pages/supported-features-sle-micro.adoc:69]: Well, the feature in SUMA to be used is the same. And we renamed the Tab to "Product Migration" which would even better match.
So we can think about renaming the description to Product Migration.

File: modules/client-configuration/pages/supported-features-sle-micro.adoc (hunk 2 above)
Comment (mcalmer) [reply modules/client-configuration/pages/supported-features-sle-micro.adoc:75]: But this "set of functionality" is also provided by salt. AFAIK there is a list at the end of this page which describe what Basic and Advanced mean.

File: modules/client-configuration/pages/supported-features-sle-micro.adoc (hunk 3 above)
Comment (mcalmer) [reply modules/client-configuration/pages/supported-features-sle-micro.adoc:132]: sure, we should let QA went over all of these items and let them explicitly test it.

File: modules/client-configuration/pages/supported-features-sle-micro.adoc [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+
+
+{star} Virtual Guest Management:
+
+In this table, virtual guest management is split into basic and advanced.
+
+Basic virtual guest management includes listing VMs, slow refresh, VM lifecycle actions (start, stop, resume, pause), and modifying VM vCPU and Memory.
+
+Advanced virtual guest management includes fast refresh, VM lifecycle actions (delete, reset, power off), modifying VM disk, network, graphical display, and graphical display configuration.
```
Comment (moio) [inline modules/client-configuration/pages/supported-features-sle-micro.adoc:165]: @cbosdo is it right to say Advanced virtual guest management includes everything in the Basic list as well?

Or are those separate?

Comment (cbosdo) [reply modules/client-configuration/pages/supported-features-sle-micro.adoc:165]: Advanced means Basic + VM console, delete, edit + Virtual nets + Virtual storage

**Comment by moio** [comment]
@keichwa this should be good now from the tech POV, can you take a look from the docs part and merge if it's OK?

**Review by mcalmer — APPROVED** [review]

**Review by keichwa — APPROVED** [review]
What a heroic effort--thanks so much, Silvio!

**Pull request merged**